exam-example-en.tex

```diff
@@ -1,7 +1,10 @@
 % Options:
-% withid: adds a header line for writing the student's ID (if a value is given, will limit the amount of boxes)
-% We recommend to keep the twoside option as it permits to add a specific header (student' name) on each new page.
-\documentclass[a4paper,twoside,withid=8]{exam}
+% withid: adds a header line for writing the student's ID (if a value is given, will limit the number of boxes)
+% anonymous: replaces student name fieds by an anonimity number (if a value is given, it specifies the number of boxes)
+%
+% We recommend to keep the twoside option as it allows to add a specific header (student' name) on each new page.
+\documentclass[a4paper,twoside,anonymous]{exam}
```

Comment:
Can you put this line as a comment and keep the previous `\documentclass[a4paper,twoside,withid=8]{exam}` as the default parameters? According to usages, most of the exams are not anon.

Reply:
Oops, my bad, I forgot to change the commented line